src/System.Net.ServicePoint/ref/System.Net.ServicePoint.cs

```diff
@@ -56,6 +56,7 @@ public enum SecurityProtocolType
     {
         SystemDefault = 0,
 #pragma warning disable CS0618
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
I copied this from the other Ssl3 obsolete attribute and was recommended by the doc-writers.
Please open a separate issue: if we want to change it, we should search and replace all other instances.